addons/payment_payulatam/tests/test_payulatam.py

```diff
@@ -13,7 +14,22 @@
 
 
 @tagged('post_install', '-at_install')
-class PayULatamTest(PayULatamCommon):
+class PayULatamTest(PayULatamCommon, PaymentHttpCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super(PayULatamTest, cls).setUpClass(chart_template_ref=chart_template_ref)
+
+        # typical data posted by payulatam after client has successfully paid
+        cls.payulatam_post_confirmation_approved_data = {
```

Comment:
Please move this in the common file. See how it's done in other payment modules.